integrationTest/integration_test.go

```diff
@@ -818,6 +818,9 @@ func makeTests(t *testing.T) []*TestGroup {
 		),
 
 		testgroup("MX",
+			not(
```

Comment:
Rather than using "not()" please add this to `loopia/auditrecords.go` (See `azuredns/auditrecords.go` as an example)